### PR TITLE
Hotfix deposit

### PR DESCRIPTION
**Pull request by EvanBarbour3**
## Bug Fixes
- Fixed `deposit` issues with decimal places

[Delivers [#127537673](https://www.pivotaltracker.com/story/show/127537673)]

### CODE REVIEW

**Comment by EvanBarbour3** [comment]
@srleach can you check over this

**Comment by srleach** [comment]
I'm happy with the code, but I think it's worth asking @sped2773 to look over the functional elements with you to ensure all the edge cases are covered. I think we've been fairly prudent with it, but safest would be to have a second pair of eyes.

**Comment by srleach** [comment]
Passing to @sped2773 and @EvanBarbour3 - Once they've approved any changes, please assign to @wnowicki

File: public/js/custom-deposit.main.js [hunk 1]
```diff
@@ -31,6 +31,20 @@ function depositValueHasChanged(changedElement){
     );
 }
 
+function depositValueWithinRange(changedElement) {
```
Comment (wnowicki) [inline public/js/custom-deposit.main.js]: 😗

**Comment by wnowicki** [comment]
@EvanBarbour3 is it ready yet?

**Comment by EvanBarbour3** [comment]
@wnowicki This is ready. Tested by @00eadeyemi and @sped2773

**Pull request merged**